pkg/hostpath/constants.go

```diff
@@ -17,7 +17,7 @@ const (
 	CSIEphemeral                       = "csi.storage.k8s.io/ephemeral"
 	SharedConfigMapShareKey            = "sharedConfigMap"
 	SharedSecretShareKey               = "sharedSecret"
-	anchorDir                          = "anchor-dir"
+	//anchorDir                          = "anchor-dir"
```

Comment:
yeah I know my earlier POC branch just commented this out, but you can feel free to just delete this